backend/src/utils/userUtils.ts

```diff
@@ -77,12 +77,16 @@ export const getUser = async (
     );
     return userResponse.body as OpenShiftUser;
   } catch (e) {
-    const error = createCustomError(
-      e.message,
-      `Error getting Oauth Info for user, ${e.response?.body?.message || e.message}`,
-      e.statusCode,
-    );
-    throw error;
+    if (isHttpError(e)) {
+      const error = createCustomError(
+        e.message,
+        `Error getting Oauth Info for user, ${errorHandler(e)}`,
+        e.statusCode,
+      );
+      throw error;
+    } else {
+      throw new Error(`Error getting Oauth Info for user, ${errorHandler(e)}`);
+    }
```

Comment:
Can probably default to status code 401:
```suggestion
    const error = createCustomError(
      e.message,
      `Error getting Oauth Info for user, ${errorHandler(e)}`,
      (isHttpError(e) && e.statusCode) || 401,
    );
    throw error;
```

Reply:
replaced `e.message` with `errorHandler(e)`, since `e` might be undefined